robotoff/app/core.py

```diff
@@ -150,6 +152,12 @@ def get_predictions(
     if value_tag:
         where_clauses.append(Prediction.value_tag == value_tag)
 
+    if brands:
+        where_clauses.append(Prediction.brands == brands)
```

Comment:
There is no brands on Prediction model, so I think you drop this brands parameter (also in the view).